### PR TITLE
Nerds

### PR DESCRIPTION
**Pull request by kjones2296**
This is Nerds' login page and email java files

### CODE REVIEW

**Review by jmonson2 — CHANGES_REQUESTED** [review]
index.html has password entered as plain text. Should be type password.

File: Emails/src/SendEmail.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import javax.mail.BodyPart;
+import javax.mail.Multipart;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMultipart;
+
+public class SendEmail {
+
+    public static void main(String[] args) {
+    		
+    	String[] arr = new String[]{"phillipphoong@gmail.com", "hunter.pierce2@knights.ucf.edu", "icebergkj@yahoo.com"};
```
Comment (kabollinger03) [inline Emails/src/SendEmail.java]: This looks like testing data. Does that need to be in master?

File: Emails/src/SendEmail.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    }
+    
+    
+    public static void sendIdividualEmail(String email) {
+    	
+        String host="smtp.gmail.com";
+        final String user="";//ENTER YOUR EMAIL!!
+        final String password=""; //ENTER YOUR PASSWORD
+
+        String to=email;
```
Comment (kabollinger03) [inline Emails/src/SendEmail.java:34]: why bother with variable assignment when you aren't doing any processing?

File: Login/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+						<img src="img/logo-fake.png" class="img-fluid pt-3 pb-5">
+					</div>
+				</div>
+				<!-- Main login form -->
+				<form class="px-5 noto" id="login-form">
+					<div class="input-group mb-3">
+					  <div class="input-group-prepend">
+					    <span class="input-group-text" id="basic-addon1"><i class="far fa-envelope"></i></span>
+					  </div>
+					  <input type="text" class="form-control-lg form-control" placeholder="Email" aria-label="Email" aria-describedby="basic-addon1">
```
Comment (kabollinger03) [inline Login/index.html:36]: type=text but it is an email. I know the HTML email validation is dumb and you are making a JS validation anyway, but....

File: Login/index.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+						</a>
+					</div>
+				</form>
+				<!-- Forgot Password form -->
+				<form class="px-5 noto" id="forgot-pw-form" style="display: none;">
+					<div class="input-group mb-3">
+					  <div class="input-group-prepend">
+					    <span class="input-group-text" id="basic-addon1"><i class="far fa-envelope"></i></span>
+					  </div>
+					  <input type="text" class="form-control-lg form-control" placeholder="Email" aria-label="Email" aria-describedby="basic-addon1">
```
Comment (kabollinger03) [inline Login/index.html:59]: See previous comment about email

File: Login/js/form.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  var y = document.getElementById("forgot-pw-form");
+  	y.style.display = "block";
+}
+
+function swap_back(){
+	var x = document.getElementById("login-form");
+	  x.style.display = "block";
+	var y = document.getElementById("forgot-pw-form");
+		y.style.display = "none";
+}
```
Comment (kabollinger03) [inline Login/js/form.js:13]: No other validation at this point? Not block worthy cause we are not dealing with web at this point but worth mentioning

**Review by kabollinger03 — COMMENTED** [review]
Mostly just some minor issues

File: Login/index.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+					</div>
+					<div class="input-group mb-2">
+					  <div class="input-group-prepend">
+					    <span class="input-group-text" id="basic-addon2"><i class="fas fa-lock"></i></span>
+					  </div>
+					  <input type="text" class="form-control-lg form-control" placeholder="Password" aria-label="Password" aria-describedby="basic-addon2">
```
Comment (kabollinger03) [inline Login/index.html:42]: password needs to be redacted! type=password

**Review by kabollinger03 — DISMISSED** [review]

File: Login/index.html (hunk 6 above)
Comment (jmonson2) [inline Login/index.html:42]: Password needs to be type="password"
```suggestion
					  <input type="password" class="form-control-lg form-control" placeholder="Password" aria-label="Password" aria-describedby="basic-addon2">
```

**Review by jmonson2 — CHANGES_REQUESTED** [review]

**Comment by kabollinger03** [comment]
Going through ZenHub... best practice is to make one PR per task. Or at least make everything in a PR relate to one story. It is ok but now IDK what story to put this PR with.

**Review by phoongp — APPROVED** [review]
Great work!

**Review by SeleneShen — APPROVED** [review]
good job

**Pull request closed without merging**